food-truck-semantic/app.py

```diff
@@ -0,0 +1,91 @@
+"""
+    app.py
+    ~~~~~~
+
+    Root of the Flask app, allowing access to spaCy
+
+    Kudos to https://spacy.io/usage/vectors-similarity for showing me
+    how to solve my similarity predicament! The Java alternatives did
+    not seem to want to work for me.
+"""
+from flask import Flask, request
+from flask_cors import CORS
+import os
+import spacy
+
+app = Flask(__name__)
+# Learned via: https://stackoverflow.com/questions/54171101/restrict-access-to-a-flask-rest-api
+cors = CORS(app, resources={"/*": {"origins": os.environ.get("FOOD_TRUCK_API")}})
```

Comment:
Would be a good idea to add this in our docker-composes's similarity service in the environment properties:
 `- FOOD_TRUCK_API=http://localhost:8080`

This is assuming those environment variables actually get passed to the app and Gunicorn doesn't just keep them to itself.

Reply:
I don't think so. This probably won't be running on the same machine as our backend, so it probably won't be able to see our main application as `localhost:8080`

Reply:
Keep in mind the docker-compose is for running locally. Personally, I'd be running this service through docker-compose instead of running it manually if I'm not making code changes to it and just using the endpoint.